mantle/generate

```diff
@@ -1,20 +1,25 @@
 #!/usr/bin/env bash
 set -eu
 
-# This script uses schematyper, which must be installed and in the PATH.
-#
 # The purpose of this script is to generate cosa/cosa_<ver>.go. Whenever
 # the schema is updated, this script should be run.
 
-hash schematyper 2>/dev/null || {
-    echo >&2 "schematyper must be installed and in the PATH to use this script: https://github.com/idubinskiy/schematyper"
-    exit 1
-}
+export GOBIN="${PWD}/bin"
```

Comment:
Let's add this to `.gitignore` in a followup?

Reply:
:+1: